css/layout.css

```diff
@@ -68,7 +103,109 @@ header {
 .intro {
   height: 30rem;
   padding-top: 4.2rem;
-  border-bottom-left-radius: 15%;
+  border-radius: 0 0 0% 100px;
 }
 
 /* intro/end */
+
+/* works/start */
+
+.works {
+  margin-top: 5.7rem;
+  margin-bottom: 5.7rem;
+}
+
+.works .wrapper {
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-row-gap: 4.4rem;
+  padding: 0;
+  width: 16.35rem;
+}
+
+/* card/start */
+
+.card {
+  flex-direction: column;
+  gap: 1rem;
+  padding: 0.8rem;
+  border: 2px solid #dfe1e6;
+  border-radius: 0.8rem;
+}
+
+.ellipse::after {
+  content: "";
+  display: inline-block;
+  width: 0.5rem;
+  height: 0.5rem;
+  margin: 0 0.6rem;
+  background-color: #c1c7d0;
+  border-radius: 50%;
+}
+
+.tags {
+  margin-top: 0.8rem;
+}
+
+.tags li {
+  flex-basis: auto;
+  margin-right: 0.6rem;
+  margin-bottom: 0;
+}
+
+.tags a {
+  padding: 0.2rem 0.6rem;
+  border-radius: 0.4rem;
+}
+
+/* card/end */
+
+/* about/start */
+
+.about {
+  height: 50rem;
+  border-top-right-radius: 5rem;
+  padding-top: 5.7rem;
+  margin-bottom: 0;
+}
+
+.about button {
+  transform: translateY(-50%);
+  margin-top: 0.8rem;
+}
+
+.arrow {
+  flex-basis: auto;
+}
+
+summary {
+  border-bottom: 1px solid #dfe1e6;
+}
+
+summary:hover {
+  cursor: pointer;
+}
+
+.lang {
+  background: #f7f7f9;
+  border-radius: 8px;
+}
+
+.lang * {
+  margin-bottom: 0;
+  padding: 0.5rem 0;
+  padding-left: 0.5rem;
+  flex-basis: auto;
+}
+
+.lang img {
+  height: 3rem;
+}
+
+.rotate {
+  transform: rotate(-90deg) translate(-70%, -100%);
+}
```

Comment:
- [ ] I believe it would be better if you edit the positioning for the icon.
- [ ] I think it would also be better if you edit the icon itself using a picture editing software.